README.md

```
@@ -2070,6 +2070,85 @@ output:
+<summary>
+Tag Browser
+</summary>
+
+### Tag Browser
+
+> This plugin is designed for internal usage in umh-core
+
+#### 1. What does this plugin do?
+
+1. For each incoming message, this plugin extracts:
+  - Topic hierarchy information (level0 through level5)
```

markdownlint-cli2 (0.17.2)

Unordered list indentation
Expected: 0; Actual: 2

(MD007, ul-indent)